samples/excel/06-named-item/create-and-remove-named-item.yaml

```diff
@@ -0,0 +1,164 @@
+id: excel-named-item-create-and-remove-named-item
+name: Create & remove a named item
+description: Create & remove a named item for a formula
+author: Microsoft
+host: EXCEL
+api_set:
+    ExcelApi: 1.4
+script:
+    content: |
+        $("#add-name").click(addName);
+        $("#remove-name").click(removeName);
+
+        async function addName() {
+            try {
+                await Excel.run(async (context) => {
+
+                    await OfficeHelpers.ExcelUtilities.forceCreateSheet(context.workbook, "Sample");
+
+                    const sheet = context.workbook.worksheets.getItem("Sample");
+
+                    let expensesTable = sheet.tables.add("A1:D1", true);
+                    expensesTable.name = "ExpensesTable";
+
+                    expensesTable.getHeaderRowRange().values = [["DATE", "MERCHANT", "CATEGORY", "AMOUNT"]];
+
+                    let newData = transactions.map(item =>
+                        [item.DATE, item.MERCHANT, item.CATEGORY, item.AMOUNT]);
+
+                    expensesTable.rows.add(null, newData);
+
+                    sheet.names.add("TotalAmount", "=SUM(ExpensesTable[AMOUNT])");
+
+                    sheet.getRange("D11").values = [["=TotalAmount"]];
+
+                    if (Office.context.requirements.isSetSupported("ExcelApi", 1.2)) {
```

Comment:
Since this sample requires ExcelApi 1.4, I don't think we need this check for 1.2